articles/5-real-life-use-cases-of-natural-language-processing-(nlp)/index.md

```diff
@@ -0,0 +1,59 @@
+Language as a method of communication helps people write, read, and speak. Individuals make plans and decisions in natural language, particularly in words. But the current artificial intelligence era is very determined to establish if individuals can communicate in a similar manner with computers. With the goal to ensure that human beings communicate with computers in their natural language, computer scientists have developed natural language processing (NLP) applications. For computers to understand the unstructured and often ambiguous human speech, they require the input of NLP applications.
+
+### Understanding Natural Language Processing
+
+The 21 st century is characterized by automation of systems and gadgets. Moreover, the use of computation is moving to elevated levels. The automation of things that we use daily is targeted to make our lives easier and better. Examples of products of automation include self-driving cars, google assistants, and artificial intelligence in systems. Each of these technologies has deep learning, machine learning or artificial intelligence technologies. Today people are embracing gadgets that rely on machine learning or artificial intelligence, due to their popularity in use in the field of data science, cloud computing, natural language processing, data engineering, computer vision, and more.
+
+Natural Language Processing (NLP) is a type of AI that seeks to enable computers to process or understand human language. Ideally, NLP does this by programming computers for analyzing and processing large quantities of natural language data. It manipulates speech, text through computational power enabled by the use of various software.
```

Comment:
You mention that NLP does this by programming computers for analyzing and processing large quantities.  It will enhance readability if you emphasize this fact. What programming techniques are you talking about? How are NLP systems designed? Since this is an article on applications of NLP, mentioning this would help the reader understand how the applications are developed.

You can mention traditional NLP techniques and deep learning techniques. How large amounts of data contain hidden information, and why traditional NLP techniques fall short? 

A good example worth mentioning is knowledge representation. How do you get patterns in a database as big as Wikipedia, which is the largest encyclopedia and multi-lingual in nature?